docs/build-css-js.md

```diff
@@ -0,0 +1,59 @@
+# Building CSS and JS
+
+This example uses `npm` and [Webpack Encore](https://symfony.com/doc/current/frontend.html).
+
+Modify `Dockerfile` to build your assets before anything else.
+
+```diff
+ARG CADDY_VERSION=2
++ARG NODE_VERSION=17
+
++FROM node:${NODE_VERSION}-alpine AS symfony_node
++WORKDIR /srv/app
++COPY package*.json ./
++RUN npm install
++## If you are building your code for production
++# RUN npm ci --only=production
++COPY . .
++RUN npm run build
+
+# "php" stage
+```
+
+Then copy the built output in `public/build` (the configuration default) to the `php` container (which will then be copied later to `caddy`).
+
+```diff
+VOLUME /srv/app/var
+
++COPY --from=symfony_node /srv/app/public/build public/build
+
+ENTRYPOINT ["docker-entrypoint"]
+CMD ["php-fpm"]
+
+FROM caddy:${CADDY_VERSION}-builder-alpine AS symfony_caddy_builder
+```
+
+Modify `docker-compose.override.yml` to add a `node` container in your development environment.
+This will provide you with hot module reloading.
+
+```diff
+  caddy:
+    volumes:
+      - ./docker/caddy/Caddyfile:/etc/caddy/Caddyfile:ro
+      - ./public:/srv/app/public:ro
+
++  node:
++    build:
++      context: .
++      target: symfony_node
++    volumes:
++      - ./:/srv/app:rw,cached
++    ports:
++     - target: 8080
++       published: 8080
++       protocol: tcp
++    command: 'sh -c "npm install; npm run dev-server -- --client-web-socket-url https://localhost:8080 --host 0.0.0.0"'
```

Comment:
```suggestion
+    command: 'sh -c "npm install; npm run dev-server -- --https --client-web-socket-url https://localhost:8080 --host 0.0.0.0"'
```

Reply:
```suggestion
+    command: 'sh -c "npm install; npm run dev-server -- --server-type https --client-web-socket-url https://localhost:8080 --host 0.0.0.0"'
```

Reply:
I tried the suggested configuration but HMR not working.
What configuration did I missed ?